src/components/TextLink.js

```diff
@@ -45,7 +49,13 @@ const TextLink = (props) => {
             href={props.href}
         >
             {({hovered, pressed}) => (
-                <Text style={[styles.link, (hovered || pressed) ? styles.linkHovered : undefined, ...additionalStyles]}>
+                <Text style={[
+                    styles.link,
+                    (hovered || pressed) ? styles.linkHovered : undefined,
+                    (hovered || pressed) ? props.hoveredStyle : undefined,
+                    ...additionalStyles,
```

Comment:
```suggestion
                <Text style={[
                    styles.link,
                    (hovered || pressed) ? styles.linkHovered : undefined,
                    ...additionalStyles,
                    (hovered || pressed) ? props.hoveredStyle : undefined,
```

The way it was before seems like it might create issues if we want to for example, change the base color via `additionalStyles` and then an optional `hoveredStyle` as it would always be overwritten by the base style.